doc/UserGuide.md

```diff
@@ -46,7 +46,7 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Persons can have any number of tags (including 0)
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
+* `add John Doe p/98765432 e/johnd@gmail.com a/123, Clementi Ave 3, #12-34, 231534`
```

Comment:
It would be nice if you indicated the format of the addressbook explicitly. How would the user know that the address has to be in this format? Example alone may be insufficient.